src/classes/CanvasRenderingContext2D.js

```diff
@@ -659,7 +648,11 @@ export default class CanvasRenderingContext2D {
   }
 
   get fillStyle() {
-    return this._fillStyleStack[this._stackIndex];
+    const color = this._fillStyleStack[this._stackIndex];
+    if(typeof color === 'string') {
+      return serializeColor(color);
+    }
+    return color;
```

Comment:
In this case, we don't need to do an `if` check if we always store the correct value on the stack.